egs/wsj/s5/utils/prepare_lang.sh

```diff
@@ -273,9 +281,9 @@ fi
 # FST due to the optional-silence [the last phone of any word gets 2 arcs].
 if [ ! -z "$unk_fst" ]; then  # if the --unk-fst option was provided...
   if "$silprob"; then
-    utils/lang/internal/modify_unk_pron.py $tmpdir/lexiconp_silprob.txt "$oov_word" || exit 1
+    utils/lang/internal/modify_unk_pron.py --boost-unk=$boost_unk $tmpdir/lexiconp_silprob.txt "$oov_word" || exit 1
```

Comment:
This will only modify the probability of the unk word in the lexicon if the --unk-fst option is provided.
I don't like that,  because  you might want to modify the unk prob even if unk is just a  normal word.
I'd prefer if you implemented this in a more generic way, not dependent on the unk-fst option being provided.